packages/forma-36-react-components/src/components/DateTime/DateTime/DateTime.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { DateTimeFormat, formatDateTime } from '../dateUtils';
+import { CoercibleDate } from '../types';
+
+export interface DateTimeProps {
+  /**
+   * The date to display as a JS Date, an ISO8601 Timestamp, or Unix Epoch Milliseconds
+   *
+   * @example '2020-08-13T01:23:45.000Z'
+   * @example new Date()
+   * @example Date.now()
+   */
+  date: CoercibleDate;
+  /**
+   * Which display format to use
+   *
+   * @default 'FULL'
+   **/
+  format?: DateTimeFormat;
+  className?: string;
+  testId?: string;
+}
+
+/**
+ * Formats a JS Date object to an absolute date and time inside of a <time /> tag
+ */
+export const DateTime: React.FC<DateTimeProps> = ({
+  date,
+  format,
+  className,
+  testId,
+}) => {
+  format = format ?? 'FULL';
```

Comment:
I think you could pass 'FULL' as default prop instead of checking if it exist or not. Like:

```jsx
export const DateTime: React.FC<DateTimeProps> = ({
  date,
  format = 'FULL',
  className,
  testId,
}) => {
// ...
}
```